experimental/javareach/cmd/reachable/main.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/google/osv-scanner/experimental/javareach"
 )
 
+const MavenDepDirPath = "META-INF/maven"
```

Comment:
nit while we're here: can we make this agnostic to platform? and use filepath.Join here?